packages/terra-date-picker/src/react-datepicker/day.jsx

```diff
@@ -206,8 +213,8 @@ class Day extends React.Component {
     const dayClassName = (this.props.dayClassName ? this.props.dayClassName(date) : undefined)
     return ['react-datepicker-day', dayClassName, 'react-datepicker-day--' + getDayOfWeekCode(this.props.day), {
       'react-datepicker-day--disabled': this.isDisabled(),
-      'react-datepicker-day--selected': this.isSameDay(this.props.selected),
-      'react-datepicker-day--selected-border': this.isSameDay(this.props.preSelection) && (document.activeElement.tagName === 'DIV' || document.activeElement === document.querySelector('[class*="previous"]')) && this.props.isCalendarKeyboardFocused,
+      'react-datepicker-day--selected': this.isSameDay(this.props.selected) && !this.isDisabled(),
+      'react-datepicker-day--selected-border': this.isSameDay(this.props.preSelection) && !isDayDisabled(this.props.selected, this.props) && (document.activeElement.tagName === 'DIV' || document.activeElement === document.querySelector('[class*="previous"]')) && this.props.isCalendarKeyboardFocused,
```

Comment:
looking at these checks, should the both be `!this.isDisabled()` or `!isDayDisabled(this.props.selected, this.props) `?

Reply:
`!this.isDisabled()` was added to prevent focus styles on current day / selected day when it was disabled. since `!this.isDisabled()` passes this.props.day and this.props.day will have current date value initially so this.props.selected  would be appropriate.  [updated](url)